stubs/asyncpg/connection.pyi

```diff
@@ -40,6 +41,7 @@ class Connection:
     def terminate(self) -> None: ...
     async def reset(self, *, timeout: Optional[float] = ...) -> None: ...
     async def reload_schema_state(self) -> None: ...
+    async def _copy_in(self, copy_stmt: str, source: AsyncIterable[bytes], timeout: int) -> None: ...
```

Comment:
I don't like this (the fact that we use it I mean, I do like the fact that we type it if we use it). But I don't see an less brittle way around it either. It does make me wonder if we should include an upper bound on the minor of the `asyncpg` constraint in `setup.py` to make sure released versions don't break on an `asyncpg` update.

Reply:
I see your concern, but adding an upper bound on the minor version doesn't provide us any guarantees, right? It only decreases the likelihood of an issue, because we narrow the range.

Reply:
You're right, I overstated the impact of that change. We would be betting on them not including big internal refactors in a patch update.

Reply:
I added the upper bound.